contact.html

```diff
@@ -24,9 +24,23 @@ <h1 class="header__title">SPACE PETS</h1>
             </nav>
         </header>
         <main>
-            <form>
-                form includes 3 inputs for name, email and message
-            </form>
+            <form
+   id="form"
+   action="https://learn-forms.netlify.com/submit/part1"
+   method="get">
+
+<label for="name">Name</label>
+<input id="name" name="name" type="text">
```

Comment:
I think if we add `require` to the input  (all of them) it avoids the form being submitted with an empty space: e.g.:
`<input id="name" name="name" type="text" require>`

Reply:
Ok this has been fixed